From f8e2829e04828329bc210d562f747577958bb12f Mon Sep 17 00:00:00 2001
From: Kexy Biscuit <kexybiscuit@aosc.io>
Date: Fri, 26 Jul 2024 17:05:57 +0800
Subject: [PATCH 29/31] chrome/browser/ui: add missing dependency

This is to fix build error due to missing dependency when we update
ninja to 1.12.
e.g. https://ci.chromium.org/ui/p/chromium/builders/build.shadow/win-build-perf-developer/1/overview

Bug: 40253918, 345645751, 346694160, 346709958, 346707816
Change-Id: I3ede8e345aef5d6d2d8bf5b155601650828bc817
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/5644627
Reviewed-by: Hidehiko Abe <hidehiko@chromium.org>
Auto-Submit: Takuto Ikuta <tikuta@chromium.org>
Commit-Queue: Takuto Ikuta <tikuta@chromium.org>
Reviewed-by: Lei Zhang <thestig@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1322655}

Co-authored-by: Takuto Ikuta <tikuta@chromium.org>
---
 chrome/browser/ui/views/side_panel/BUILD.gn | 14 +++++++++++++-
 1 file changed, 13 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
index dcdc90aa85..e292b30fde 100644
--- a/chrome/browser/ui/views/side_panel/BUILD.gn
+++ b/chrome/browser/ui/views/side_panel/BUILD.gn
@@ -223,12 +223,24 @@ static_library("side_panel") {
     "//ui/gfx/geometry:geometry_skia",
     "//ui/webui",
 
-    # TODO(crbug.com/346711540,crbug.com/346699817,crbug.com/345645751):
+    # TODO(crbug.com/345645751): Use //chrome/browser instead after fixing cyclic dependency.
+    "//components/enterprise/common/proto:connectors_proto",
+
+    # TODO(crbug.com/346711540,crbug.com/346699817,crbug.com/345645751,
+    #      crbug.com/346694160,crbug.com/346709958,crbug.com/346707816):
     # Use //chrome/browser/ui instead after fixing cyclic dependency.
+    "//chrome/browser/cart:mojo_bindings",
+    "//chrome/browser/companion/visual_query",
     "//chrome/browser/ui:webui_name_variants",
+    "//chrome/browser/ui/webui/side_panel/bookmarks:mojo_bindings",
+    "//chrome/browser/ui/webui/side_panel/customize_chrome:mojo_bindings",
+    "//chrome/browser/ui/webui/side_panel/reading_list:mojo_bindings",
     "//components/enterprise/buildflags",
+    "//components/page_image_service/mojom:mojo_bindings",
     "//components/paint_preview/buildflags",
     "//components/segmentation_platform/public/proto",
+    "//components/user_education/webui",
     "//components/webapps/common:mojo_bindings",
+    "//ui/webui/resources/cr_components/commerce:mojo_bindings",
   ]
 }
-- 
2.45.2

